src/commands/util/ping.mts

```diff
@@ -0,0 +1,12 @@
+// Basic ping bot
+
+import { SlashCommandBuilder } from "discord.js";
+import { Command } from "../../types.mjs";
```

Comment:
I might be wrong, but I'm pretty sure you can do this (and should be preferred if you can).
```suggestion
import type { Command } from "../../types";
```

Reply:
Seems like TS doesn't like it when you're doing that